docs/design/autoware-interfaces/components/interfaces-future.md

```diff
@@ -0,0 +1,66 @@
+---
+components:
+  - adapi
+  - control
+  - localization
+  - map
+  - perception
+  - planning
+  - sensing
+  - simulator
+  - system
+  - vehicle
+
+interfaces:
+  - name: /localization/initialization_state
+    type: autoware_adapi_v1_msgs/msg/LocalizationInitializationState
```

Comment:
What about making it independent of autoware_adapi_v1_msgs?
I think it would be better to close the interaction between components to component interface msgs only.